apollo-build-maven-extensions/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"  encoding="UTF-8"?>
+<!--
+  ~ Copyright 2023 Apollo Authors
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<parent>
+		<groupId>com.ctrip.framework.apollo</groupId>
+		<artifactId>apollo</artifactId>
+		<version>${revision}</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>apollo-build-maven-extensions</artifactId>
+	<name>Apollo Build Maven Extensions</name>
+
+
+	<profiles>
+		<profile>
+			<id>sql-convert</id>
+			<activation>
+				<activeByDefault>false</activeByDefault>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-compiler-plugin</artifactId>
+					</plugin>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>exec-maven-plugin</artifactId>
+						<version>3.0.0</version>
+						<executions>
+							<execution>
+								<id>sql-convert</id>
+								<phase>compile</phase>
+								<goals>
+									<goal>java</goal>
+								</goals>
+							</execution>
+						</executions>
+						<configuration>
+							<mainClass>com.ctrip.framework.apollo.maven.extensions.ApolloAssemblySqlConverter</mainClass>
```

Comment:
This looks interesting. Do the SQL files under `sql/assembly` folder update automatically?

Reply:
BTW, I met an error trying to create an app:

```
com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Expected name at line 1 column 3 path $[0].
	at com.google.gson.Gson.fromJson(Gson.java:1241)
	at com.google.gson.Gson.fromJson(Gson.java:1137)
	at com.google.gson.Gson.fromJson(Gson.java:1047)
	at com.google.gson.Gson.fromJson(Gson.java:1014)
	at com.ctrip.framework.apollo.portal.component.config.PortalConfig.organizations(PortalConfig.java:175)
	at com.ctrip.framework.apollo.portal.controller.OrganizationController.loadOrganization(OrganizationController.java:43)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
```

It looks like the `organizations` value is different in the h2 database:

<img width="695" alt="image" src="https://github.com/apolloconfig/apollo/assets/837658/8a0c5351-bf41-40bc-b2cc-c746b099d53d">

While the record in the mysql database is:

<img width="785" alt="image" src="https://github.com/apolloconfig/apollo/assets/837658/66f1d9f8-36e8-489a-83e8-e151af890078">

Reply:
fixed